postreise/plot/plot_utilization_map.py

```diff
@@ -35,92 +20,109 @@ def map_risk_bind(
 
     branch_congestion = branch_congestion[branch_congestion[risk_or_bind] > 0]
     branch_congestion.sort_values(by=[risk_or_bind])
     branch_map = project_branch(branch_congestion)
     min_val = branch_congestion[risk_or_bind].min() if vmin is None else vmin
     max_val = branch_congestion[risk_or_bind].max() if vmax is None else vmax
     mapper = linear_cmap(
-        field_name=risk_or_bind, palette=traffic_palette, low=min_val, high=max_val
-    )
-    color_bar = ColorBar(
-        color_mapper=mapper["transform"],
-        width=385 if is_website else 500,
-        height=5,
-        location=(0, 0),
-        title=risk_or_bind_units,
-        orientation="horizontal",
-        padding=5,
+        field_name=risk_or_bind, palette=palette, low=min_val, high=max_val
     )
     multi_line_source = ColumnDataSource(
         {
             "xs": branch_map[["from_x", "to_x"]].values.tolist(),
             "ys": branch_map[["from_y", "to_y"]].values.tolist(),
             risk_or_bind: branch_map[risk_or_bind],
             "value": branch_map[risk_or_bind].round(),
-            "cap": branch_map["capacity"] / 1000 + 2,
+            "cap": (
+                branch_map["capacity"] * select_branch_scale_factor
+                + select_branch_min_width
+            ),
             "capacity": branch_map.rateA.round(),
         }
     )
 
     # Set up figure
     p = figure(
-        tools=tools,
+        tools="pan,wheel_zoom,reset,save",
         x_axis_location=None,
         y_axis_location=None,
-        plot_width=800,
-        plot_height=800,
+        plot_width=figsize[0],
+        plot_height=figsize[1],
         output_backend="webgl",
-        sizing_mode="stretch_both",
         match_aspect=True,
     )
-    p.add_layout(color_bar, "center")
-    p.add_tile(get_provider(Vendors.CARTODBPOSITRON))
-    p.patches(a, b, fill_alpha=0.0, line_color="gray", line_width=1)
+    if show_color_bar:
+        color_bar = ColorBar(
+            color_mapper=mapper["transform"],
+            width=385 if is_website else 500,
+            height=5,
+            location=(0, 0),
+            title=risk_or_bind_units,
+            orientation="horizontal",
+            padding=5,
+        )
+        p.add_layout(color_bar, "center")
+    default_plot_states_kwargs = {"fill_alpha": 0.0, "background_map": True}
+    if plot_states_kwargs is not None:
+        all_plot_states_kwargs = default_plot_states_kwargs.update(**plot_states_kwargs)
+    else:
+        all_plot_states_kwargs = default_plot_states_kwargs
+    plot_states(bokeh_figure=p, **all_plot_states_kwargs)
     p.multi_line(
-        "xs",
-        "ys",
+        branch_map_all[["from_x", "to_x"]].to_numpy().tolist(),
+        branch_map_all[["from_y", "to_y"]].to_numpy().tolist(),
         color="gray",
-        line_width="cap",
-        source=multi_line_source_all,
+        line_width=(
+            branch_map_all["rateA"] * all_branch_scale_factor + all_branch_min_width
```

Comment:
Same comment here about scaling (`all_branch_scale_factor` default value is 5e-4), we could normalize `"rateA"` to have a more natural scaling factor (around 1)

Reply:
If we do the same translation from `pixels/MW` to `pixels/GW` here, then we end up with a default scaling factor of 0.5 to match what the code is currently doing. I would not want to change the default behavior here without buy-in from the website team first.

Reply:
Done.